src/app/FormList.jsx

```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router';
 import MdDelete from 'react-icons/lib/md/delete';
 import _ from 'lodash';
 
-import { deleteForm, listForms } from 'forms/FormActions';
+import { deleteForm, fetchForms } from 'forms/FormActions';
```

Comment:
I like how components can import just the actions they need from a local actions file very much.  Very clear pattern showing what you're brining in here, where it's coming from -> what the cross concept dependency situation may be.